src/completions.ts

```diff
@@ -296,40 +289,37 @@ export default class Completions implements CompletionItemProvider
 
         let search = document.getText(match);
 
-        let potential = this.getTags().filter((tag) => {
+        let potential = this.tags.filter((tag) => {
             return tag.tag.match(search) !== null;
         });
 
         potential.forEach(tag => {
+            let snippet = this.replaceTagSnippet(tag.tag, tag.snippet);
+
             let item = new CompletionItem(tag.tag, CompletionItemKind.Snippet);
             item.range = match;
-            item.insertText = new SnippetString(tag.snippet);
+            item.insertText = new SnippetString(snippet);
 
             result.push(item);
         });
 
         return result;
     }
-
+    
     /**
-     * Get the tag list for completions
-     *
-     * @returns {Array<{tag:string, snippet:string}>}
+     * Replace tag
+     * 
+     * @param {string} tag 
+     * @param {string} snippet 
+     * @returns {string}
      */
-    protected getTags():Array<{tag:string, snippet:string}>
+    protected replaceTagSnippet(tag:string, snippet:string): string
     {
-        if (!this.formatted) {
```

Comment:
If it is marked as formatted, it cannot take effect immediately after the user modifies the author information